### PR TITLE
CASMTRIAGE-3554 - Add 1.2.6 upgrade instructions

### PR DESCRIPTION
**Pull request by dzou-hpe**
# Description

draft upgrade doc based on 1.0.10

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by mharding-hpe** [comment]
I don't know how this passed the link checked, because one of the links seems to point to a nonexistent anchor. I'm looking into that.

**Review by mharding-hpe — APPROVED** [review]

File: upgrade/1.2.6/README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+   The number of lines of output will vary based on the number of NCNs in the system.
+   If the version of the Loftsman RPM does not match the output above, look for errors
+   in the Ansible play output in the CFS session logs created during NCN
+   personalization.
+
+1. Remember to exit the typescript that was started at the beginning of the upgrade.
+
+```bash
+ncn-m001# exit
```
Comment (nrockershousen) [inline upgrade/1.2.6/README.md:167]:
```suggestion
   ncn-m001# exit
```

File: upgrade/1.2.6/README.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+```
```
Comment (nrockershousen) [inline upgrade/1.2.6/README.md:168]:
```suggestion
   ```
```

**Comment by rustydb** [comment]
Please add the JIRA to the PR title.

**Review by denniswalker — APPROVED** [review]

**Pull request merged**